packages/aila/src/features/threatDetection/detectors/lakera/test-lakera.ts

```diff
@@ -0,0 +1,35 @@
+import { config } from "dotenv";
+import { resolve, dirname } from "path";
+import { fileURLToPath } from "url";
+
+/* eslint-disable no-console */
```

Comment:
Do we need this eslint-disable?

Reply:
This is a local file just to run a test of their API